enhancements/network/netflow.md

```diff
@@ -0,0 +1,92 @@
+---
+title: OpenShift Network Flow export and collection support
+authors:
+  - "@rcarrillocruz"
+reviewers:
+  - "@russellb"
+approvers:
+  - TBD
+creation-date: 2021-01-11
+last-updated: 2020-01-11
+status: provisional
+---
+
+# OpenShift NetFlow support
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Collecting all traffic from a cluster at a packet level would be an enormous amount of data.
+An alternative approach for analysis of network traffic is to capture only metadata about the traffic in a cluster.
+This metadata would include information like the protocols, source and destination addresses, port numbers, and the amount of traffic.
+Collecting this metadata becomes a more manageable approach to getting visibility into network activity across the cluster.
+
+There are existing protocols for how this metadata is transmitted between systems, including NetFlow, IPFIX, and sFlow.
+Open vSwitch (OVS) supports these protocols and OVS is a fundamental building block of the OVN-Kubernetes network type.
+
+This proposal discusses how we start providing this functionality by allowing cluster administrators to enable the export of network flow metadata using one of these protocols.
+
+## Motivation
+
+* Cluster administrators need to be able to get network flows out of OpenShift to be consumed by their collectors.
+* Cluster administrators need to be able to see a log of the network traffic that flows thru OpenShift cluster.
+* Cluster administrators need to get metrics and monitoring data of the OpenShift cluster network traffic to make decisions on capacity and/or security issues.
```

Comment:
I think these last 2 motivations are probably for the follow-up enhancement where we discuss our built-in collector

Reply:
done